### ISSUE TITLE
[Geometry] Redundant vacuum materials should be deleted

### ISSUE THREAD

**Issue by cvuosalo** [cms-sw/cmssw#33738]
The following elementary materials defined in geometry XML files are redundant:
```
CTPPS_Diamond_Materials:PPS_Timing_Primary_Vacuum
CTPPS_Diamond_Materials:PPS_Timing_Secondary_Vacuum
RP_Materials:RP_Primary_Vacuum
RP_Materials:RP_Secondary_Vacuum
zdcmaterials:Galactic
```
The vacua are all identical and should be replaced with the standard `materials:Vacuum`. `zdcmaterials:Galactic` is unused and should be deleted.

**Comment by cvuosalo** [comment]
assign geometry

**Comment by civanch** [comment]
@bsunanda , @cvuosalo , I suspect this is already fixed?

**Comment by bsunanda** [comment]
I think this is done
________________________________

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
It seems zdcmaterials:Galactic is still there:
https://github.com/cms-sw/cmssw/blob/master/Geometry/ForwardCommonData/data/zdcmaterials/2021/v1/zdcmaterials.xml#L4

**Comment by bsunanda** [comment]
I just submitted a PR to remove that material which was never used
________________________________

**Comment by qliphy** [comment]
https://github.com/cms-sw/cmssw/pull/34064

**Comment by cvuosalo** [comment]
No, sorry, this issue is not completed. The following vacua still exist:
```
CTPPS_Diamond_Materials:PPS_Timing_Primary_Vacuum
CTPPS_Diamond_Materials:PPS_Timing_Secondary_Vacuum
RP_Materials:RP_Primary_Vacuum
RP_Materials:RP_Secondary_Vacuum
```
It is not a big problem, but it would be nice to replace them all with the standard `materials:Vacuum`. Unfortunately, there are several dozen files that use them. Maybe the old versions would be left in place.

**Comment by civanch** [comment]
Fixed in #34101

**Comment by cvuosalo** [comment]
+1
